BankingSystem/bank/CentralBank.java

```diff
@@ -0,0 +1,97 @@
+package BankingSystem.bank;
+
+import BankingSystem.account.Account;
+import BankingSystem.account.EAccount;
+import BankingSystem.utility.Dialog;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class CentralBank {
+
+
+    ArrayList<Account> accountList;
+    HashMap<EAccount, InterestCalculator> interestCalculatorTable;
+
+//    HashMap<Integer, Bank> bankTable;
+
+    public CentralBank(){
+        accountList = new ArrayList<>();
+        interestCalculatorTable = new HashMap<>();
+        interestCalculatorTable.put(EAccount.N, new NormalInterestCalculator());
+        interestCalculatorTable.put(EAccount.S, new SaveInterestCalculator());
+    }
+    public boolean registerAccount(Account newAccount){
+        if(!checkVerifyNum(newAccount.getAccountNum())) {
+            Dialog.systemMsg("등록실패!! 유효하지 않은 계좌번호입니다.");
+            return false;
+        }
+        for (Account account : accountList){
+            if(account.getAccountNum().equals(newAccount.getAccountNum())) {
+                Dialog.systemMsg("등록실패!! 중복된 계좌번호입니다.");
+                return false;
+            }
+        }
+        accountList.add(newAccount);
+        return true;
+    }
+
+    public boolean checkVerifyNum(String accountNum) {
```

Comment:
해당 클래스 안에서만 사용하는 메소드라면 접근 제어자의 단계를 높이는 것이 나을 것 같습니다!
```suggestion
    private boolean checkVerifyNum(String accountNum) {
```